blog/2023-06-06-scaling-bitcoin-to-billions.md

```diff
@@ -0,0 +1,85 @@
+---
+slug: scale-btc-to-billions
+title: Scale Bitcoin to billions of users
+authors: [philipp]
+tags: [scaling, lightning, bitcoin]
+---
+
+# How to scale to billions of users
+
+The recent mempool situation reminded us all about the scarcity of blocksize.
+The mempool is full, transaction fees are soaring, which makes small on-chain transaction not viable anymore.
+This is a problem as we can't scale Bitcoin to billions of users.
+
+![](/2023-06-06-scaling-btc/2023-06-06-future-city.png)
+
+<!-- truncate -->
+
+## Why Lightning in the first place
+
+The [Lightning Network](http://lightning.network/) comes to resscue: it is a layer-2 built on top of Bitcoin which facilitates near instant transactions for a very low fee (for now).
+It interacts with the main chain if required but primarily, payments are settled “off-chain”.
+
+## The problem
+
+Exactly these two transactions can be a problem for many users: if the price to open a channel costs $10 and closing the channel costs again $10, then small channels do not make sense economically.
+Users are incentiviced to only open big channels and keep them open as long as possible.
+
+Additionally, running a Lightning Node needs
+
+- high capital upfront; and
+- technical expertise
+
+Requirements that will be unrealistic for some, especially those who only wish to receive funds with low friction.
+
+## Why not custodial Lightning channels?
+
+The simplest way to interact with the Lightning Network is to use a custodial wallet; this could be through an app like the [Wallet of Satoshi](https://www.walletofsatoshi.com/) or an exchange account you might have with a provider like CashApp.
```

Comment:
Is it the simplest way though? I'd argue that Phoenix and Breez are also very simple to ineract with the Lightning Network. What we need though are LSPs and inbound liquidity, that makes it complicated.